understanding/22/consistent-help.html

```diff
@@ -155,8 +155,22 @@ <h3>Failures for Consistent Help</h3>
             </li>
          </ul>
       </section>
-
    </section>
+   
+      <section id="resources">
+        <h2>Resources</h2>
+
+        <ul>
+            <li><a href="https://rawgit.com/w3c/coga/master/gap-analysis/#table6">Cognitive Accessibility Gap Analysis Topic 6: Familiar Interface</a></li>
```

Comment:
link to https://w3c.github.io/coga/gap-analysis/#table6 or https://www.w3.org/TR/coga-gap-analysis/#table6